src/main/java/ticketservice/TicketServiceApplicationContext.java

```diff
@@ -4,21 +4,25 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
 
 import javax.sql.DataSource;
 import java.io.IOException;
 import java.util.Properties;
 
 @Configuration
 @ComponentScan
+@EnableTransactionManagement
 public class TicketServiceApplicationContext {
     @Bean
     public DataSource dataSource() {
         Properties properties = new Properties();
         PGSimpleDataSource pgSimpleDataSource = new PGSimpleDataSource();
         try {
             properties.load(
-                    Thread.currentThread().getContextClassLoader().getResourceAsStream("db_connection.properties"));
+                    Thread.currentThread().getContextClassLoader().getResourceAsStream("application.properties"));
```

Comment:
do not load properties with the help of `Thread`. Use 1 of `Spring` approaches:
1. add `@PropertySource("classpath:application.properties")`
   and read values with `@Value`:
```java
 @Value("${postgres.url}")
 private String postgresUrl;
   ... 
   pgSimpleDataSource.setUrl(postgresUrl);
   ...
 ```

2. add `@PropertySource("classpath:application.properties")`
   and read values with `Spring Environment`:
```java
private final Environment env;
public TicketServiceApplicationContext(Environment env) {this.env = env;}
   ... 
   pgSimpleDataSource.setUrl(env.getProperty("postgres.url"));
   ...
 ```

Reply:
Fixed in https://github.com/kolosokvit/ticket-service/pull/7/commits/36637aa4b2eb2e1605c331b5a7d108b399c5378b